### ISSUE TITLE
Imcompatible with the latest Doctrine 2.10

### ISSUE THREAD

**Issue by hantsy** [ramsey/uuid-doctrine#168]
The latest Doctrine 2.10 was updated to dbal 3.x which is incompatible with this `uuid-doctrine`.

When installing this in a Symfony project, it failed. 

```bash 
  Problem 1
    - Root composer.json requires ramsey/uuid-doctrine ^1.7 -> satisfiable by ramsey/uuid-doctrine[1.7.0].
    - ramsey/uuid-doctrine 1.7.0 requires doctrine/dbal ^2.5 -> found doctrine/dbal[v2.5.0, ..., 2.13.4] but the package is fixed to 3.1.3 (lock file version) by a partial update and t
hat version does not match. Make sure you list it as an argument for the update command.
```

**Comment by Ocramius** [comment]
This is not a bug: the supported dependency range is declared by this package by design.

The only missing bit is a release of `1.8.0`, which will bring in `doctrine/dbal:^3` support as per https://github.com/ramsey/uuid-doctrine/pull/166 and https://github.com/ramsey/uuid-doctrine/commit/706f1d1a7880536f60de6f918299326540b19001

**Comment by ramsey** [comment]
I've just released 1.8.0, which includes support for `doctrine/dbal:^3`. Thanks!
